scripts/autoplacement_24_in_96.py

```diff
@@ -1,72 +1,18 @@
 #!/usr/bin/env python
-import sys
 
-from EPPs.common import StepEPP
+from EPPs.common import Autoplacement
 
 
 # Script for performing autoplacement of samples. If 1 input and 1 output then 1:1 placement. If multiple input plates then
 # takes all samples from each plate before next plate Loops through all inputs and assigns them to the next available space
 # by column-row in the output plate
```

Comment:
Can you put these comments in the class docstring:
```python
class Autoplacement24in96(Autoplacement):
    """
    Script for performing autoplacement of samples. If 1 input and 1 output then 1:1 placement. If multiple input plates then
    takes all samples from each plate before next plate Loops through all inputs and assigns them to the next available space
    by column-row in the output plate
    """"

Reply:
updated

Reply:
updated